tests/actor/test_battery_pool.py

```diff
@@ -0,0 +1,929 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+"""Tests for BatteryPool module."""
+
+import logging
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from typing import Callable, List, Optional, Set
+
+import frequenz.api.microgrid.microgrid_pb2 as microgrid_pb
+import pytz
+import time_machine
+from frequenz.api.microgrid.battery_pb2 import Battery as PbBatteryData
```

Comment:
I think these should be aliased just as `PbBattery` for consistency,  because there's `from ...battery_pb2 import Data as PbBatteryData` somewhere else.

Reply:
Done